src/Microsoft.DotNet.Interactive/FormattedValue.cs

```diff
@@ -19,35 +19,39 @@ public FormattedValue(string mimeType, string value)
 
         MimeType = mimeType;
         Value = value;
+        SuppressDisplay = suppressDisplay;
     }
 
     public string MimeType { get; }
 
     public string Value { get; }
 
-    public static FormattedValue CreateSingleFromObject(object value, string mimeType = null)
+    public bool SuppressDisplay { get; }
+
+    public static FormattedValue CreateSingleFromObject(object value, string mimeType = null, bool suppressDisplay = false)
     {
         if (mimeType is null)
         {
             mimeType = Formatter.GetPreferredMimeTypesFor(value?.GetType()).First();
         }
 
-        return new FormattedValue(mimeType, value.ToDisplayString(mimeType));
+        return new FormattedValue(mimeType, value.ToDisplayString(mimeType), suppressDisplay);
     }
 
     public static IReadOnlyList<FormattedValue> CreateManyFromObject(object value, params string[] mimeTypes)
+        => CreateManyFromObject(value, suppressDisplay: false, mimeTypes);
+
+    public static IReadOnlyList<FormattedValue> CreateManyFromObject(object value, bool suppressDisplay = false, params string[] mimeTypes)
```

Comment:
Once we start having this many properties on the type, I'm not sure the immutability is helping any, and adding the overload is a lot more surface area than for example just making `FormattedValue.SuppressDisplay` settable.

Reply:
Good call. Removed the overloads and made the property settable in the latest commit.